transcom/mymove
# MB-16800: Add feature flags capability

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16800)

## Summary

This provides APIs for accessing and using feature flags. It does not implement any features that use feature flags.

This PR supersedes #10970 which has some examples of possible usage.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

### How to test

If `FEATURE_FLAG_SERVER_URL` is undefined, the code will look for environment variables. This is a handy way to test locally without running the feature flag server. See `featureflag.EnvFetcher` for more details.

To test with flipt, do
1. Start the client: `make client_run`
2. Start the server: `FEATURE_FLAG_SERVER_URL=http://localhost:9080 make server_run`
3. Start flipt: `make feature_flag_docker`

1. Log into the admin app at `http://adminlocal:3000`
2. Go to `Evaluate Feature Flag`
3. Enter `multi_variant` in the search field and hit `Evaluate`
4. See the `Value` is `two`


### Frontend


### Backend
